Source/ARTCrypto.h

```diff
@@ -44,11 +51,30 @@ NS_ASSUME_NONNULL_BEGIN
 
 @end
 
+/**
+ The ARTCrypto object ensures that message payloads are encrypted, can never be decrypted by Ably, and can only be decrypted by other clients that share the same secret symmetric key.
+ */
 @interface ARTCrypto : NSObject
 
-+ (ARTCipherParams *)getDefaultParams:(NSDictionary *)values;
+/**
+ Retrieves, or optionally sets, the `ARTCipherParams` for the channel.
+ @param params Overrides the default parameters. A suitable key must be provided as a minimum.
+ @return `ARTCipherParams` A ARTCipherParams object, using the default values for any field not supplied.
+ */
++ (ARTCipherParams *)getDefaultParams:(NSDictionary *)params;
+
+/**
+ Generates a random key to be used in the encryption of the channel.
+ @param keyLength The length of the key, in bits, to be generated. If not specified, this is equal to the default keyLength of the default algorithm: for AES this is 256 bits.
+ @return `NSData` The key as a binary data.
+ */
++ (NSData *)generateRandomKey:(NSUInteger)keyLength;
+
+/**
+ Same as `+getDefaultParams:`, but with the default algorithm and key length: AES, 256 bits.
```

Comment:
The description for `generateRandomKey` is the following in the canonical table:

```
Generates a random key to be used in the encryption of the channel.
```

I'm assuming this should be something like "Generates a random key use for encrypting message payloads." instead given @lawrence-forooghian 's comment on `getDefaultParams`.

Reply:
But we have two methods with this name: one with parameter `keyLength`, and one without parameters. Canonical table contains only one description and notes that "_If not specified, this is equal to the default keyLength of the default algorithm: for AES this is 256 bits._". So I decided to produce different description for method without parameter. Also I've messed up with the referred method: `+getDefaultParams:` instead of `generateRandomKey:`. Duh! @lawrence-forooghian @m-hulbert

Reply:
https://github.com/ably/ably-cocoa/pull/1439/commits/3cce6135c1776c6802ab5a640dd0331fc46b0894